spatio_temporal_voxel_layer/include/spatio_temporal_voxel_layer/spatio_temporal_voxel_grid.hpp

```diff
@@ -130,15 +130,16 @@ class SpatioTemporalVoxelGrid
     rclcpp::Clock::SharedPtr clock,
     const float & voxel_size, const double & background_value,
     const int & decay_model, const double & voxel_decay,
-    const bool & pub_voxels);
+    const bool & pub_voxels, const double & voxel_distance_decay);
```

Comment:
Pedantic: but ordering this next to `voxel_decay` would be great to group like-things